openssh.yaml

```diff
@@ -100,6 +102,17 @@ subpackages:
           mkdir -p "${{targets.subpkgdir}}"/usr/lib/ssh
           mv "${{targets.destdir}}"/usr/lib/ssh/sftp-server "${{targets.subpkgdir}}"/usr/lib/ssh/
 
+  - name: "openssh-server-pam"
```

Comment:
This can be `openssh-pam` since PAM stands for Pluggable Authentication Modules, which is what this is intended to be used as.